### PR TITLE
change dependency version to file ^6.0.0

### PR DESCRIPTION
**Pull request by kaynz**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <repo>https://git-wip-us.apache.org/repos/asf/cordova-plugin-file-transfer.git</repo>
     <issue>https://issues.apache.org/jira/browse/CB/component/12320650</issue>
 
-    <dependency id="cordova-plugin-file" version="^5.0.0" />
+    <dependency id="cordova-plugin-file" version="^6.0.0" />
```
Comment (macdonst) [inline plugin.xml]: @kaynz I think it would be better to make it ">5.0.0".

Comment (kaynz) [reply plugin.xml]: @macdonst Yep, I agree.

Comment (macdonst) [reply plugin.xml]: @kaynz sorry, should be ">=5.0.0" not just ">"

Comment (fikriauliya) [reply plugin.xml]: Shouldn't we also update `version="1.7.1-dev"`?

Comment (kaynz) [reply plugin.xml]: @macdonst @fikriauliya done

**Comment by oleksiy-nesterov** [comment]
I am also waiting for this changes...  PhoneGap Build can't build an app without this stuff.

**Comment by davidgv88** [comment]
+1! Please merge!

**Comment by macdonst** [comment]
Merged but it might be a bit before a new release is out as we did a plugin release recently. As well this plugin is deprecated. Folks should start spending time replacing this plugin with XHR2 which should be available everywhere now.

**Comment by kaynz** [comment]
@macdonst The main problem in migrating to XHR2 is that it's not that easy to replace this plugin. The migration guide is incomplete:

https://cordova.apache.org/blog/2017/10/18/from-filetransfer-to-xhr2.html

> TL;DR
> Standards are great and all, but what do you actually have to copy-paste to replace the previous FileTransfer examples? We have you covered:

> Here's a replacement for FileTransfer's "Download a Binary File" example:

It's not a replacement at all. Old FileTransfer is saving the binary on the filesystem, where it survives a restart of the application. With the replacement code it will only be in the memory.

In my opinion, it would make more sense to replace relevant native code in this plugin with the corresponding XHR2 functions and keep this plugin up.
